style.css

```diff
@@ -1,3 +1,300 @@
\ No newline at end of file
+body {
+    background-color: rgb(136, 136, 187);
+    overflow: scroll;
+    background-color: #383838;
+  }
+  
+  #policeman {
+    position: flex;
+    text-align: center;
+    padding-top: 50px;
+    font-size: 50px;
+    font-weight: bold;
+    color: bisque;
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+  }
+  
+  .link {
+    font-size: 60px;
+    font-weight: bold;
+    color: rgb(92, 137, 234);
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+  }
+    
+  #background-video {
+    height: 100vh;
+    width: 100vw;
+    object-fit: cover;
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 0;
+    bottom: 0;
+    z-index: -1;
+  }
+  
+  .heading {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+  }
+  
+  #mad-libs {
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+    font-size: 120px;
+    color: bisque;
+    margin-top: 10px;
+    margin-bottom: 5px;
+  }
+  
+  #first-heading {
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+    font-size: 70px;
+    color: beige;
+    display: flex;
+    text-align: center;
+    justify-content: center;
+    border: solid rgb(201, 124, 124);
+    border-radius: 10px;
+    box-shadow: 1px 1px 2px white, 0 0 25px rgb(197, 192, 255),
+      0 0 25px rgba(32, 28, 153, 0.911);
+    padding-left: 100px;
+    padding-right: 100px;
+    margin-top: 10px;
+  }
+  
+  .flex-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    gap: 80px;
+  }
+  
+  #edit {
+    font-size: 60px;
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+    color: cornsilk;
+    display: flex;
+    justify-content: center;
+    margin-top: 0px;
+  }
+  
+  .madLibsEdit {
+    background-color: rgba(160, 73, 73, 0.397);
+    border: solid pink;
+    border-radius: 10px;
+    box-shadow: 1px 1px 2px white, 0 0 25px pink, 0 0 25px red;
+    width: 40%;
+    padding: 50px;
+  }
+  
+  #preview {
+    font-size: 60px;
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+    color: cornsilk;
+    display: flex;
+    justify-content: center;
+    margin-top: 0px;
+  }
+  
+  .madLibsPreview {
+    background-color: rgba(29, 47, 206, 0.278);
+    border: solid pink;
+    border-radius: 10px;
+    box-shadow: 1px 1px 2px white, 0 0 25px pink, 0 0 25px red;
+    width: 40%;
+    padding: 50px;
+  }
+  
+  input {
+    width: 50px;
+    background-color: rgb(152, 168, 229);
+    border-radius: 8px;
+    border-color: white;
+    border: solid pink;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-family: "Merriweather Sans", sans-serif;
+    font-family: "Shalimar", cursive;
+    font-size: 30px;
+    color: azure;
+  }

```

Comment:
Try to refrain as much as possible from targeting HTML elements by tags in CSS. Instead, use classes and assign them to the inputs you want to change.